apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -1,65 +1,419 @@
 <template>
-  <v-navigation-drawer
-    :value="value"
-    :clipped="$vuetify.breakpoint.lgAndUp"
-    app
-    :style="{'margin-top': classification ? '5em' : '3.5em', 'z-index': 11}"
-    disable-resize-watcher
-    disable-route-watcher
-    fixed
-    temporary
-    width="600px"
-    @input="$emit('input', $event)"
-    @blur="value = false"
-  >
-    <v-expansion-panels v-model="active_path" accordion>
-      <DropdownContent
-        header-text="Results"
-        :files="visible_evaluation_files"
-        :all-selected="all_evaluations_selected"
-        :enable-compare-view="true"
-        :compare-view-active="compareViewActive"
-        @toggle-all="toggle_all_evaluations"
-        @toggle-compare-view="compareView"
-        @changed-files="$emit('changed-files')"
-      />
-      <DropdownContent
-        header-text="Profiles"
-        :files="visible_profile_files"
-        :all-selected="all_profiles_selected"
-        @toggle-all="toggle_all_profiles"
-        @changed-files="$emit('changed-files')"
-      />
-    </v-expansion-panels>
-  </v-navigation-drawer>
+  <div>
+    <ChecklistTargetDataModal
+      :visible="showTargetModal"
+      style="display: none"
+      @close-modal="showTargetModal = false"
+    />
+    <ChecklistTechnologyAreaModal
+      :visible="showTechnologyModal"
+      style="display: none"
+      @close-modal="showTechnologyModal = false"
+    />
+
+    <v-navigation-drawer
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11}"
+      disable-resize-watcher
+      disable-route-watcher
+      fixed
+      permanent
+      width="45px"
+      @input="$emit('input', $event)"
+      @blur="value = false"
+    >
+      <v-container v-if="!isSideUtilityDrawerShown" fill-height fluid>
+        <v-row align="center" justify="center">
+          <v-col
+            ><v-icon
```

Comment:
check to make sure that any of these '><' things aren't there since they look really dumb lol.  i think it's caused by the tags all being in one line and the linter decides to break up the lines weirdly.  (i.e. <v-col><v-icon> turns into <v-col \n ><v-icon> instead of <v-col> \n <v-icon> which is what we want)

Reply:
Went through and fixed the ones that I saw